runtime/Python3/src/antlr4/__init__.py

```diff
@@ -1,21 +1,21 @@
-from antlr4.Token import Token
-from antlr4.InputStream import InputStream
-from antlr4.FileStream import FileStream
-from antlr4.StdinStream import StdinStream
-from antlr4.BufferedTokenStream import TokenStream
-from antlr4.CommonTokenStream import CommonTokenStream
-from antlr4.Lexer import Lexer
-from antlr4.Parser import Parser
-from antlr4.dfa.DFA import DFA
-from antlr4.atn.ATN import ATN
-from antlr4.atn.ATNDeserializer import ATNDeserializer
-from antlr4.atn.LexerATNSimulator import LexerATNSimulator
-from antlr4.atn.ParserATNSimulator import ParserATNSimulator
-from antlr4.atn.PredictionMode import PredictionMode
-from antlr4.PredictionContext import PredictionContextCache
-from antlr4.ParserRuleContext import RuleContext, ParserRuleContext
-from antlr4.tree.Tree import ParseTreeListener, ParseTreeVisitor, ParseTreeWalker, TerminalNode, ErrorNode, RuleNode
-from antlr4.error.Errors import RecognitionException, IllegalStateException, NoViableAltException
-from antlr4.error.ErrorStrategy import BailErrorStrategy
-from antlr4.error.DiagnosticErrorListener import DiagnosticErrorListener
-from antlr4.Utils import str_list
+from .Token import Token
```

Comment:
In the future I would recommend making these explicit export based imports, as mentioned in https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-no-implicit-reexport.